### ISSUE TITLE
NoSectionError when running CUPP

### ISSUE THREAD

**Issue by Kaos-Industries** [Mebus/cupp#16]
I'm running CUPP on Windows using Cygwin (`bash` terminal). I have it installed at `C:\cupp`, and have added it to the `PATH` environment variable. When doing `cupp.py` in bash, I get the following error: 

```
Traceback (most recent call last):
  File "/cygdrive/c/cupp/cupp.py", line 50, in <module>
    years = config.get('years', 'years').split(',')
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'years'
```

By the looks of it, it may have something to do with the configuration file `cupp.cfg`, but that's all I can tell.

**Comment by 5P4D3** [comment]
im getting this fucking error to.

**Comment by Mebus** [comment]
I just tested this on Cygwin64 and I do not get this error. Please tell me how to reproduce this.

**Comment by 5P4D3** [comment]
cupp3 works fine for me :)

**Comment by anandumdas** [comment]
Same error.

My setup is as follows:
Ubuntu 16.04, 64 bit. Running bash.

1. Cloned the repo
2. Added the path to PATH variable in .profile
3. Source .profile
4. Run cupp.py from a directory other that /cupp/ 

Error apperars: 
```
$cupp.py 
Traceback (most recent call last):
  File "/home/anandu/Downloads/cupp/cupp.py", line 50, in <module>
    years = config.get('years', 'years').split(',')
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'years'

```
No error appears when ran from the parent `cupp` directory. 
cupp3.py works without error.

**Comment by anandumdas** [comment]
I have checked my cupp.cfg and years is there in the file, uncommented. Please see the snippet

```
# If you don't need some chars, just comment it! Duplicates are allowed too.
# For adding chars, you might need to add some lines in cupp.py...
[years]
years=2008,2009,2010,2011,2012,2013,2014,2015,2016

[leet]
a=4
i=1

```

**Comment by flying8lack** [comment]
what python version are you using, seems that the error is not from the config, because it is normal.

**Comment by anandumdas** [comment]
```
$ python --version
Python 2.7.12

```
I have python 3 also. cupp3 works fine with python3. But cupp has this error when running with 2.7

**Comment by anandumdas** [comment]
@Mebus can you please try running cupp from a directory other than cupp's directory after setting up proper PATHs. I am on Ubuntu 16, python 2.7.

**Comment by Mebus** [comment]
@anandumdas I fixed this with 40f8654492c80880702a01616d132540ad6f0af8 

I have converted the script to Python3, too.
